pycufsm/fsm.py

```diff
@@ -333,45 +334,43 @@ def strip(
         # INTRODUCE CONSTRAINTS AND REDUCE k_global MATRICES TO FREE PARTS ONLY
         k_global_ff = r_matrix.transpose() @ k_global @ r_matrix
         kg_global_ff = r_matrix.transpose() @ kg_global @ r_matrix
-
         # SOLVE THE EIGENVALUE PROBLEM
         # Determine which solver to use
         # small problems usually use eig (dense matrix),
         # and large problems use eigs (sparse matrix).
         # the eigs solver is not as stable as the full eig solver...
         # LAPACK reciprocal condition estimator
-        # rcond_num = 1 / np.linalg.cond(np.linalg.pinv(kg_global_ff) @ k_global_ff)
+        rcond_num = 1 / np.linalg.cond(np.linalg.pinv(kg_global_ff) @ k_global_ff)
 
         # Here, assume when rcond_num is bigger than half of the eps, eigs can provide
         # reliable solution. Otherwise, eig, the robust solver should be used.
-        # if rcond_num >= np.spacing(1.0) / 2:
-        #     eig_sparse = True
-        #     # eigs
-        # else:
-        #     eig_sparse = False
-        #     # eig
+        if rcond_num >= np.spacing(1.0) / 2:
+            eig_sparse = True
+            # eigs
+        else:
+            eig_sparse = False
+            # eig
```

Comment:
The eig_sparse variable still isn't used for anything. Leave this commented out.

Reply:
Will do